resources/testharness.js

```diff
@@ -792,7 +792,10 @@ policies and contribution forms [3].
 
     function done() {
         if (tests.tests.length === 0) {
-            tests.set_file_is_test();
+            tests.status.status = tests.status.ERROR;
+            tests.status.message = "The test signaled completion before testing began.";
```

Comment:
Can this be more explicit about what was and wasn’t called? Like “done() was called without first defining any tests”

Reply:
You got it